docs/tbdev_getting_started.ipynb

```diff
@@ -1,36 +1,25 @@
 {
```

Comment:
<p>Put <code>tensorboard dev delete --experiment_id pQpJNh00RG2Lf1zOe9BrQA </code> around pretty quotes for code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/2992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>